build.gradle

```diff
@@ -552,6 +551,7 @@ if (HUDSON_JOB_NAME == "not_hudson") {
 defineProperty("RELEASE_SUFFIX", relSuffix)
 defineProperty("RELEASE_VERSION_SHORT", "${RELEASE_VERSION}${RELEASE_SUFFIX}")
 defineProperty("RELEASE_VERSION_LONG", "${RELEASE_VERSION_SHORT}+${PROMOTED_BUILD_NUMBER}${relOpt}")
+defineProperty("MAVEN_VERSION", "$RELEASE_VERSION_LONG")
```

Comment:
This will be "11-ea+19" for the next build. Will this work? Or does it need to be something more like "11.0.0-ea-19"?

Here are the settings that were used by our Hudson job for build 18:

```
PROMOTED_BUILD_NUMBER: 18
RELEASE_VERSION: 11
RELEASE_SUFFIX: -ea
RELEASE_VERSION_SHORT: 11-ea
RELEASE_VERSION_LONG: 11-ea+18
RELEASE_VERSION_PADDED: 11.0.0.0
```